spec/index.html

```diff
@@ -1233,7 +1245,7 @@ <h3>Grammar</h3>
           '<code class="grammar-literal">true</code>',
           '<code class="grammar-literal">false</code>')
           are case-sensitive.
-          Keywords in double quotes
+          Keywords in quotation marks
           ("<code class="grammar-literal">BASE</code>",
           "<code class="grammar-literal">PREFIX</code>")
           are case-insensitive.
```

Comment:
This is not correct. The grammar only has "" words.

It was right in RDF 1.1 - it isn't in RDF 1.2.

Reply:
I'll double-check the grammar and the HTML version to make sure the use of accent vs quotation mark is consistent with 1.1.

Reply:
I updated the HTML EBNF so that use of `"` and `'` matches that in the raw EBNF version. We're not consistent, for example, on use of `"."` vs `'.'`. I corrected errors for `'a'`, `'true'`, and `'false'`. `"PREFIX"` and `"BASE"` are correct as is. The other cases for non alphabetic characters doesn't really matter other than for editorial consistency.